include/cetl/unbounded_variant.hpp

```diff
@@ -961,6 +961,57 @@ struct base_move<Pmr, Footprint, Copyable, true /*Movable*/, Alignment>
 
 }  // namespace detail
 
+// MARK: - "In Place" nested types.
+
+/// Namespace to nest `unbounded_variant` specific declarations.
+///
+namespace ub_var
+{
+
+/// Implementation similar to \ref std::in_place_type_t or \ref cetl::pf17::in_place_type_t.
+///
+/// In use by several \ref cetl::unbounded_variant constructors.
+///
+/// Can't use directly either of already existing `std::in_place_type_t` or `cetl::pf17::in_place_type_t` types due to
+/// C++14 limitation and polyfill optionality (by design in CETL, according to Scott), so a bit of code duplication.
+///
+template <typename T>
+struct in_place_type_t
```

Comment:
Can this be a public member of unbounded_variant? `ub_var` feels like an ugly hack.

Reply:
ok